src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -478,6 +479,16 @@ event::ptr primitive_inst::realloc_if_needed() {
     return ev;
 }
 
+bool primitive_inst::use_async_compilation() {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_IF(debug_config->disable_async_compilation) {
+        return false;
+    }
+    return (_node->is_type<convolution>() ||
+            _node->is_type<fully_connected>() ||
```

Comment:
btw, shouldn't gemm be a part of this list?

Reply:
My test results didn't show no benefit when `gemm` is added to the async compilation queue. I'll test it again with a long running scenario such as the `softmax` case.

Reply:
I updated the async compilation queue to include `gemm`.